BigANNBinaryEmbeddingOnlyScearioBase.cs

```diff
@@ -294,8 +296,10 @@ protected async Task PerformQuery(bool isWarmup, int numQueries, int KVal, strin
 
         private QueryDefinition ConstructQueryDefinition(int K, float[] queryVector)
         {
-            string queryText = $"SELECT TOP {K} c.id, VectorDistance(c.{this.EmbeddingColumn}, @vectorEmbedding) AS similarityScore " +
-                $"FROM c ORDER BY VectorDistance(c.{this.EmbeddingColumn}, @vectorEmbedding, false)";
+            int searchListSizeMultiplier = Convert.ToInt32(this.Configurations["AppSettings:scenario:searchListSizeMultiplier"]);
+            string obj_expr = $"{{ 'searchListSizeMultiplier': {searchListSizeMultiplier} }}";

```

Comment:
In future we may change the multiplier from 5 to a different value, so if appSetting defines 0, let us not pass any value and keep the original value to pick what backend has as default value at that time.